src/lib/components/right-option/RightOption.svelte

```diff
@@ -0,0 +1,101 @@
+<script lang="ts">
+  export let xAxis = 0;
+  export let yAxis = 0;
+  export let menuItems: Array<{
+    onClick: () => void;
+    displayText: string;
+    disabled: boolean;
+  }> = [];
+  export let noOfRows = 0;
+  export let noOfColumns = 0;
+  let mouseX = 0;
+  let mouseY = 0;
+
+  const calculateRightOptionHeight = () => {
+    if (noOfRows === 5) {
+      return [200, 180];
```

Comment:
set base values and multiply or add to get the desired height to make the function generic instead of if else

Reply:
resolved